result/bgpprefix.yaml

```diff
@@ -281,3 +303,340 @@ components:
+        An extended community is an 8-bytes value. It is divided into two main parts. The first 2 bytes of the community  encode a type and optonal sub-type field.
+        The last 6 bytes (or 7 bytes for types without a sub-type) carry a unique set of data in a format defined by the type and optional sub-type field. 
+        Extended communities provide a larger  range for grouping or categorizing communities.
+      type: object
+      properties:        
+        choice:
+          type: string             
+          x-enum:
+            transitive_2octet_as_type:
+              x-field-uid: 1
+            transitive_ipv4_address_type:
+              x-field-uid: 2
+            transitive_4octet_as_type:
+              x-field-uid: 3
+            transitive_opaque_type:
+              x-field-uid: 4         
+            non_transitive_2octet_as_type:
+              x-field-uid: 5         
+          x-field-uid: 1
+        transitive_2octet_as_type:        
+          x-field-uid: 2
+          $ref: '#/components/schemas/BgpPrefix.ExtendedCommunity.Transitive2OctetAsType'
+        transitive_ipv4_address_type:        
+          x-field-uid: 3
+          $ref: '#/components/schemas/BgpPrefix.ExtendedCommunity.TransitiveIpv4AddressType'
+        transitive_4octet_as_type:        
+          x-field-uid: 4
+          $ref: '#/components/schemas/BgpPrefix.ExtendedCommunity.Transitive4OctetAsType'
+        transitive_opaque_type:        
+          x-field-uid: 5
+          $ref: '#/components/schemas/BgpPrefix.ExtendedCommunity.TransitiveOpaqueType'
+        non_transitive_2octet_as_type:        
+          x-field-uid: 6
+          $ref: '#/components/schemas/BgpPrefix.ExtendedCommunity.NonTransitive2OctetAsType'
+    BgpPrefix.ExtendedCommunity.Transitive2OctetAsType.RouteTarget:
+      description: >-
+        The Route Target Community identifies one or more routers that may receive a set of routes (that carry this Community) carried by BGP. 
+        It is sent with sub-type as 0x02.            
+      type: object
+      properties:
+        global_2byte_as:
+          description: >-
+            The two octet IANA assigned AS value assigned to the Autonomous System.
+          type: integer
+          format: uint32
+          maximum: 65535
+          x-field-uid: 1
+        local_4byte_admin:
+          type: integer
+          format: uint32
+          description: >-
+            The Local Administrator sub-field contains a number from a numbering space that is administered by the organization to which the
+            Autonomous System number carried in the Global Administrator sub-field has been assigned by an appropriate authority.
+          x-field-uid: 2
+    BgpPrefix.ExtendedCommunity.Transitive2OctetAsType.RouteOrigin:
+      description: >-          
+        The Route Origin Community identifies one or more routers that inject a set of routes (that carry this Community) into BGP.
```

Comment:
same as above

Reply:
In this case, it is not referring to BGP Update  but injection of routes into BGP itself based on match wit Route Origin value. So kept as is.